prepare_data.py

```diff
@@ -89,6 +88,8 @@ def main():
                     help="validation split ration")
     ap.add_argument("--cores", type=int, help='Number of virtual cores to parallelize over',
                     default=max(1, (mp.cpu_count() - 2) // 2)) # To prevent IPC issues
+    ap.add_argument("--ocr_engine", type=str, default='pytesseract',
+                    help="OCR used to extract text: [pytesseract/aws_textract]")
```

Comment:
Thoughts on using the `choices` keyword within argument parser? I believe that the help message will show the choices and it's just another way to error prune.

```suggestion
                    help='OCR used to extract text', choices=['pytesseract', 'aws_textract]')
```